test/mocks/providers/provider_ranking.dart

```diff
@@ -0,0 +1,19 @@
+import "package:fake_cloud_firestore/fake_cloud_firestore.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/services/database/firestore_service.dart";
+import "package:proxima/viewmodels/login_view_model.dart";
+
+import "../data/firestore_user.dart";
+
+Future<ProviderContainer> rankingProviderContainer(
+  FakeFirebaseFirestore fakeFirebaseFirestore,
+) async {
+  await setUserFirestore(fakeFirebaseFirestore, testingUserFirestore);
+
+  return ProviderContainer(
+    overrides: [
+      firestoreProvider.overrideWithValue(fakeFirebaseFirestore),
+      validLoggedInUserIdProvider.overrideWith((ref) => testingUserFirestoreId),
```

Comment:
I made this comment since we had a PR which one of the roles was to fix this kind of heterogeneities, hence my comment. I communicated to the rest of the team and we will try to avoid this kind of review comment in the future reviews, thank you! :)